src/main/java/sg/ncl/MainController.java

```diff
@@ -144,39 +145,121 @@ public String tools() {
-            response.setContentType("application/force-download");
-            response.setHeader("Content-Disposition", "attachment; filename=future_plan.pdf");
+            stream = getClass().getClassLoader().getResourceAsStream("downloads/order_form.pdf");
+            response.setContentType(APPLICATION_FORCE_DOWNLOAD);
+            response.setHeader(CONTENT_DISPOSITION, "attachment; filename=order_form.pdf");
             IOUtils.copy(stream, response.getOutputStream());
             response.flushBuffer();
-        } catch (Exception ex) {
-            logger.info("Error writing file to output stream.");
-            throw new FuturePlanDownloadException("IOError writing file to output stream");
+        } catch (IOException ex) {
+            logger.info("Error for download orderform.");
+            throw new OrderFormDownloadException("Error for download orderform.");
         } finally {
             if (stream != null) {
                 stream.close();
             }
         }
     }
 
-    @RequestMapping(value="/orderform/download", method=RequestMethod.GET)
-    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException, IOException {
+    @RequestMapping(value="/SubscriptionAgreement/download", method=RequestMethod.GET)
+    public void subscriptionAgreementDownload(HttpServletResponse response) throws MasterSubscriptionAgreementDownloadException, IOException {
         InputStream stream = null;
-        response.setContentType("application/pdf");
+        response.setContentType(MediaType.APPLICATION_PDF_VALUE);
         try {
-            stream = getClass().getClassLoader().getResourceAsStream("downloads/order_form.pdf");
-            response.setContentType("application/force-download");
-            response.setHeader("Content-Disposition", "attachment; filename=order_form.pdf");
+            stream = getClass().getClassLoader().getResourceAsStream("downloads/SubscriptionAgreement.pdf");
+            response.setContentType(APPLICATION_FORCE_DOWNLOAD);
+            response.setHeader(CONTENT_DISPOSITION, "attachment; filename=SubscriptionAgreement.pdf");
             IOUtils.copy(stream, response.getOutputStream());
             response.flushBuffer();
         } catch (IOException ex) {
-            logger.info("Error writing file to output stream.");
-            throw new OrderFormDownloadException("IOError writing file to output stream");
+            logger.info("Error for subscription download." + ex.getMessage());
+            throw new MasterSubscriptionAgreementDownloadException("Error for subscription download.");
+        } finally {
+            if (stream != null) {
+                stream.close();
+            }
+        }
+    }
+
+    @RequestMapping(value="/UsagePolicy/download", method=RequestMethod.GET)
+    public void usagePolicyDownload(HttpServletResponse response) throws UsagePolicyDownloadException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: sg.ncl.exceptions.UsagePolicyDownloadException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)